### DISCUSSION TITLE
Error reading IdSSLIOHandlerSocketOpenSSL1.SSLOptions.Method: Invalid property value

### DISCUSSION THREAD

**Discussion by llovvoll** [IndySockets/Indy#606] [Q&A] [answered]
Hi everyone, I followed (https://web.archive.org/web/20200925081341/http://ww2.indyproject.org/Sockets/Docs/Indy10Installation.EN.aspx) to install Indy 10 on Delphi XE2 , it seems to be installed successfully, but an error will occur when I want to use TLS1.2. I wonder if I have done something wrong? If you have any suggestions, thank you

Application Error

Exception EReadError in module SalaryMail1.exe at 0004D104.

Error reading IdSSLIOHandlerSocketOpenSSL1.SSLOptions.Method: Invalid property value.

**Comment by rlebeau** [comment, marked as answer]
Just FYI, the install instructions are also now available in Indy's Wiki here on GitHub:
https://github.com/IndySockets/Indy/wiki/Updating-Indy

As for the error, XE2 was released in 2011, but Indy did not get support for TLS 1.1 and 1.2 until 2012, so it makes me think you probably still have some old Indy files somewhere on your dev machine that are being found during compilation instead of the updated files.

**Comment by llovvoll** [comment]
> Just FYI, the install instructions are also now available in Indy's Wiki here on GitHub: https://github.com/IndySockets/Indy/wiki/Updating-Indy
> 
> As for the error, XE2 was released in 2011, but Indy did not get support for TLS 1.1 and 1.2 until 2012, so it makes me think you probably still have some old Indy files somewhere on your dev machine that are being found during compilation instead of the updated files.

Thank you so much, problem solved
